app/javascript/packs/check-support-ft-unlock.ts

```diff
@@ -0,0 +1,14 @@
+import { isWebauthnPlatformAuthenticatorAvailable } from '@18f/identity-webauthn';
+
+async function checkSupportFtUnlock() {
+  let platformauthenticatorAvailableInput = (document.getElementById('platform_authenticator_available') as HTMLInputElement)
```

Comment:
should we have this bail if the element is not there for some reason?
```suggestion
  let platformauthenticatorAvailableInput = (document.getElementById('platform_authenticator_available') as HTMLInputElement)
  if (!platformauthenticatorAvailableInput) {
    return;
  }
```

Reply:
Or alternatively, would it make sense to use our component/custom element approach here? so the JS gets activated when the custom element is in the DOM, which would minimize the risk of the JS running on a page without the element it needs?